.circleci/config.yml

```diff
@@ -41,14 +43,14 @@ jobs:
           -DCMAKE_SHARED_LINKER_FLAGS="$LD_FLAGS"
           -DCMAKE_INSTALL_PREFIX:PATH="$PWD/_install"
           -DENABLE_STATIC=OFF
-          -DTRACE=ON
           -DMUST_BUILD_TOXAV=ON
       - run: cd c-toxcore/_build && ninja install -j$(nproc)
-#     - run:
-#         export CFLAGS="-I$PWD/c-toxcore/_install/include -fsanitize=address";
-#         export LDFLAGS="-L$PWD/c-toxcore/_install/lib -Wl,-rpath,$PWD/c-toxcore/_install/lib";
-#         python3 setup.py build_ext --inplace
-#     - run: ASAN_OPTIONS=detect_leaks=0
-#         LD_PRELOAD=libasan.so.5
-#         PYTHONPATH=.
-#         python3 tests/tests.py
+      - run: |
+          export CFLAGS="-I$PWD/c-toxcore/_install/include -fsanitize=address"
```

Comment:
in most situations, `-fsanitize=address` is a linker flag. why is it done so funny here?